packages/@apphosting/common/src/index.ts

```diff
@@ -1,5 +1,58 @@
 import { spawn } from "child_process";
 
+// Output bundle specifications to be written to bundle.yaml
+export interface OutputBundle {
+  version: "v1alpha";
+  serverConfig: ServerConfig;
+  metadata: Metadata;
+}
+
+// Fields needed to configure the App Hosting server
+interface ServerConfig {
+  // Command to start the server (e.g. ["node", "dist/index.js"]) assume this command is run from the root dir of the workspace
+  runCommand: string[];
```

Comment:
ooc why string[] and not string?

Reply:
Hmm okay I personally would recommend we prioritize whatever makes for a better DX for adapter authors here. Splitting by " " isn't a ton of overhead on the buildpacks side, but adapter authors will have to pay the cost of consuming this API every time they add a new framework

Reply:
Sure that makes sense, changed this to string